docs/content/dagster-cloud/deployment/agents/customizing-configuration.mdx

```diff
@@ -94,3 +94,43 @@ compute_logs:
   module: dagster.core.storage.noop_compute_log_manager
   class: NoOpComputeLogManager
 ```
+
+---
+
+## Streaming compute logs to AWS S3
```

Comment:
Maybe explain the motivation here up top? something like: "You can also stream compute logs to an S3 bucket in your cluster without having to provide Dagster Cloud with your S3 credentials".

It might not be totally clear from this text that the "streaming" part of this (upload_interval) can be configured separately from the "write to s3" part of this - maybe just call this "Writing compute logs to S3" and have a separate note about how to make it stream vs. writing at the end of the step?